docs/zone-pivot-groups.yml

```diff
@@ -8,3 +8,11 @@ groups:
       title: Visual Studio 2022
     - id: dotnet-cli
       title: .NET CLI
+- id: devices-set-one
+  title: Devices
+  prompt: Choose your deployment device
+  pivots:
```

Comment:
Android will need to go first, as it's the default build experience (this may change - I'll take it up offline with you).